publication-commons/src/test/java/no/unit/nva/publication/doi/dto/PublicationTest.java

```diff
@@ -0,0 +1,47 @@
+package no.unit.nva.publication.doi.dto;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.is;
+
+import java.net.URI;
+import java.util.List;
+import no.unit.nva.publication.doi.dto.Publication.PublicationBuilder;
+import org.junit.jupiter.api.Test;
+
+class PublicationTest {
```

Comment:
I think it would be worth to have a test where we create the DTO's from the expected serialized format (DynamoDB stream records)

Reply:
Thats another task: https://unit.atlassian.net/browse/NP-1704 

Where we will make a mapper to construct the DTO from the Dynamodb stream record.